Meetings/SG/20200806.md

```diff
@@ -0,0 +1,85 @@
+# ETF Steering Group (ETF SG) - 24th meeting
+
+## Logistics
+
+Location: Webex
+
+Date/Time: 06/08/2020, 10:00-11:30
+
+## Attendance
+
+- SG members:
+  - JRC: Marco Minghini
+  - ii: Clemens Portele (main), Jon Herrmann
+
+- Observers:
+  - Guadaltel: Carlos Palma
+
+## Minutes of previous meeting and open actions
+
+- [Minutes by ii](https://github.com/etf-validator/governance/blob/master/Meetings/SG/20200616.md) accepted and merged.
+- All open actions are on the agenda, except those on
+  - [EIP #76](https://github.com/etf-validator/governance/issues/76)
+  - [issue #208](https://github.com/etf-validator/etf-webapp/issues/208#issuecomment-619811910)
+
+where there are no news. These should be re-discussed at the next meeting in case there are news.
+
+## EIPs for SG discussion
+
+- [EIP #59](https://github.com/etf-validator/governance/issues/59)
+
+  - the module is already developed and tested, and the idea is to include it in the next ETF release.
+  - it is agreed to flag the EIP as _EIP-approved_ and move it under _TC: scheduled for release_ in the [EIP project board](https://github.com/orgs/etf-validator/projects/2). The EIP will be included in the next release of the ETF.
+  - [ ] ii to move the repository under the etf organisation.
+
+
+- [EIP #57](https://github.com/etf-validator/governance/issues/57)
+
+  - it is agreed to flag the EIP as _EIP-approved_ and move it under _TC: scheduled for release_ in the [EIP project board](https://github.com/orgs/etf-validator/projects/2). The EIP will be included in the next release of the ETF.
+  - [ ] ii to integrate the new v. 2 of the library in the ETF BaseX test driver.
+  - the old version (v. 1) of the GmlGeoX module needs to be integrated with the new version of deegree, and this should be done in time for the next release.
+  - [x] Contractors to create a [new EIP](https://github.com/etf-validator/governance/issues/96) to explain the approach.
+  - once the approach is proven to work for v. 1 of GmlGeoX, in the future the same approach can be extended to v. 2 (again by first discussing the approach in a new EIP).

```

Comment:
I made a note that we wanted to open 2 EIPs. One for v1 (which we did) and one for V2, which will be just a draft for now.

Additionally a Pull Request for the v1 change should be created in the [etf-gmlgeox repo](https://github.com/etf-validator/etf-gmlgeox/pulls).